### PR TITLE
Handles file links in activity log

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/13623

## Testing
**URL to test:** Local (bc its a branch from a branch)

**Steps to test:**
1. Add a new file by filling in the link field instead of uploading a file
2. In the file table, edit the file link.
3. Confirm that the edit is captured in the activity log

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js [hunk 1]
```diff
@@ -568,4 +573,3 @@ export const ProjectActivityLogTableMaps = {
     },
   },
 };
-
```
Comment (johnclary) [inline moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js:571]: there were two ne lines; now there is one.

**Comment by johnclary** [comment]
this is my first activity log PR 😊

**Review by mddilley — APPROVED** [review]
Tested this locally with the `Moped Local Editor User Login`, and I see:
Screenshot 2023-08-31 at 11 55 47 AM

Achievement unlocked! 🙏

**Review by chiaberry — APPROVED** [review]
Screenshot 2023-08-31 at 12 10 05 PM

yay! and since I still have prod data locally, I could get a preview of prod data

**Comment by johnclary** [comment]
thank you!

**Pull request merged**